src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/InitialConsiderationHelperTest.java

```diff
@@ -0,0 +1,154 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.DateListedTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.JurCodesTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.DateListedType;
+import uk.gov.hmcts.et.common.model.ccd.types.JurCodesType;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class InitialConsiderationHelperTest {

```

Comment:
Don't need tests to be public.  Same applies for the individual test methods too

Reply:
done